docker-compose-mysql.yml

```diff
@@ -52,10 +52,7 @@ services:
       - ZOOKEEPER_CONNECTION_TIMEOUT_MS=60000
 
   mysql:
-    build:
-      context: ./mysql
-      args:
-        EVENTUATE_COMMON_VERSION: "$EVENTUATE_COMMON_VERSION"
+    image: eventuateio/eventuate-tram-sagas-mysql:0.13.0.RELEASE
```

Comment:
`0.13.0.RELEASE` should be `$eventuateTramSagasImageVersion`

Reply:
Fixed